pkgs/tools/misc/coreutils/default.nix

```diff
@@ -29,7 +29,9 @@ stdenv.mkDerivation rec {
          # To be removed in coreutils-8.32.
          ++ optional stdenv.hostPlatform.isMusl ./avoid-false-positive-in-date-debug-test.patch
          # Fix compilation in musl-cross environments. To be removed in coreutils-8.32.
-         ++ optional stdenv.hostPlatform.isMusl ./coreutils-8.31-musl-cross.patch;
+         ++ optional stdenv.hostPlatform.isMusl ./coreutils-8.31-musl-cross.patch
+         # Fix compilation in android-cross environments. To be removed in coreutils-8.32.
+         ++ optional stdenv.hostPlatform.isAndroid ./coreutils-8.31-android-cross.patch;
```

Comment:
If that patch also works on other platforms I rather would include it unconditionally to avoid bitrot.

Reply:
Done